reference/mv3/intro/mv3-migration/content-scripts/popup.js

```diff
@@ -0,0 +1,31 @@
+let injectFunction = document.getElementById('inject-function');
+let injectFile = document.getElementById('inject-file');
+
+async function getCurrentTab() {
+  let queryOptions = { active: true, currentWindow: true };
+  let [tab] = await chrome.tabs.query(queryOptions);
+  return tab;
+}
+
+function showAlert() {
+  alert('Function test alert');
+}
+
+injectFunction.addEventListener('click', async () => {
+  let tab = await getCurrentTab();
+
+  chrome.scripting.executeScript({
+    target: {tabId: tab.id},
+    function: showAlert
```

Comment:
wait—didn't this get renamed to `func`?

Reply:
Yep, missed that the rename landed in stable.